src/Map/src/Renderer/AbstractRenderer.php

```diff
@@ -90,8 +92,11 @@ private function getMapAttributes(Map $map): array
 
         foreach ($attrs['markers'] as $key => $marker) {
             $attrs['markers'][$key]['@id'] = $computeId($marker);
+            if (null !== $this->uxIconRenderer && null !== $marker['icon'] && 'ux-icon' === $marker['icon']['type']) {
+                $attrs['markers'][$key]['icon']['content'] = $this->uxIconRenderer->render($marker['icon']['content']);
+                $attrs['markers'][$key]['icon']['type'] = 'inline-svg';
+            }
```

Comment:
I believe we want to throw an error if the user wanted to use an `UxIcon` but without symfony/ux-icons, wdyt?

```suggestion
            if (UxIcon::TYPE === ($marker['icon']['type'] ?? null)) {
                if (!$this->uxIconRenderer) {
                    throw new ...; 
                }
                
                $attrs['markers'][$key]['icon']['content'] = $this->uxIconRenderer->render($marker['icon']['content']);
                $attrs['markers'][$key]['icon']['type'] = InlineSvg::TYPE;
            }
```

Reply:
(and it requires to add `TYPE` constants to InlineSvg, UxIcon, etc.. classes, we don't want to hardcode them)

Reply:
We already throw a similar exception in UxIconRenderer, it might be redundant ?

Reply:
But so, `$uxIconRenderer` property does not need to be nullable right?